components/compliance-service/api/tests/export_test.go

```diff
@@ -306,10 +306,17 @@ func TestJSONExportWithTwoControlFiltersReturnsError(t *testing.T) {
 	stream, err := reporting.Export(context.Background(), &profileFilterQuery)
 	require.NoError(t, err)
 
-	_, err = stream.Recv()
+	reports, err := getJSONReportsFromStream(stream)
+	require.NoError(t, err)
 
-	grpctest.AssertCode(t, codes.InvalidArgument, err)
-	assert.Equal(t, "rpc error: code = InvalidArgument desc = Invalid: Only one 'control' filter is allowed", err.Error())
+	report := reports[0]
```

Comment:
We probably want a `require.Len` above this, otherwise if this returns 0 results it'll crash the test run.